### ISSUE TITLE
Some comments on the code

### ISSUE THREAD

**Issue by gongzhitaao** [lilyxy/Haiven#9]
# TL;DR Interface First

Since you are working separately on different components of the app, I have a few comments on the integration.

## Backend/Frontend

I don't see any REST API implemented in the `route.py` yet, you probably want to design the API with the frontend together, e.g., deciding on the message format, etc.  After you have this interface worked out, the frontend and backend may be developed and tested separately.  It might be easier for integration.

For example, the server could have an anchor point: `/upload_audio`, which expects a JSON object

```json
{
  "timestamp": "202007161536",
  "audio": "/path/to/your/wav/file/",
}
```

So for the backend, you could use `curl` to debug the server locally without the frontend.  And for the frontend, you only need to make sure the JSON is properly prepared.  Then at least in theory, the integration will be smooth.

## Audio Processing

You may create a placeholder function, e.g., 

```py
from enum import IntEnum


class Sentiment(IntEnum):
  HAPPY = 1
  SAD = 2
  ANGRY = 3


def detect_sentiment(audio_file: str):
  return Sentiment.HAPPY
```

The backend could assume this function is ready and just call this function in the server.  And whoever works on this detection may update this function without worrying about the other parts of the system.

Great work!
